partner_coc/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# pylint: disable=missing-docstring,protected-access
+from openerp.osv import orm, fields
+
+
+class ResPartner(orm.Model):
+    _inherit = 'res.partner'
+
+    _columns = {
+        'coc_registration_number': fields.function(
+            lambda self, *args, **kwargs:
+            self._compute_identification(*args, **kwargs),
+            arg='coc',
+            fnct_inv=lambda self, *args, **kwargs:
+            self._inverse_identification(*args, **kwargs),
+            fnct_inv_arg='coc',
+            type='char',
+            fnct_search=lambda self, *args, **kwargs:
+            self._search_identification(*args, **kwargs),
+            method=True, store=True, readonly=False,
```

Comment:
does v7 call the inverse function for stored function fields? v10 does not.

Reply:
In v7 apparently it does, otherwise this would not have worked at all.

Anyways, prove is here:
```
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/bisdom/bisdom_breda_crm/model/res_partner.py(209)write()
-> cr, uid, ids, values, context=context)
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/trp_oe_staging/partner_address/model/res_partner.py(270)write()
-> context=context)
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/partner-contact/partner_relations/model/res_partner.py(293)write()
-> cr, uid, ids, vals, context=self._update_context(context, ids))
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/odoo/openerp/addons/base/res/res_partner.py(536)write()
-> result = super(res_partner,self).write(cr, uid, ids, vals, context=context)
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/odoo/addons/mail/mail_thread.py(297)write()
-> result = super(mail_thread, self).write(cr, uid, ids, values, context=context)
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/odoo/openerp/osv/orm.py(4339)write()
-> result += self._columns[field].set(cr, self, id, field, vals[field], user, context=rel_context) or []
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/odoo/openerp/osv/fields.py(1182)set()
-> self._fnct_inv(obj, cr, user, id, name, value, self._fnct_inv_arg, context)
  /home/openeyedev/var/projects/bisdom_7001/odoo/parts/partner-contact/partner_coc/models/res_partner.py(16)<lambda>()
-> self._inverse_identification(*args, **kwargs),
> /home/openeyedev/var/projects/bisdom_7001/odoo/parts/partner-contact/partner_identification/models/res_partner.py(104)_inverse_identification()
-> if not field_name or not field_value:
(Pdb)
```